### PR TITLE
Feature/pelagos 3531 fix css for admin only template

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

**Comment by praneethpr** [comment]
Do you want to merge base branch into this branch? I don't think it is up to date

File: src/Pelagos/Bundle/AppBundle/Resources/views/layout.ui.html.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        add_js(
+            [
+                '//cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.11.1/jquery.validate.min.js',
+                '//cdnjs.cloudflare.com/ajax/libs/jquery-noty/2.3.5/packaged/jquery.noty.packaged.min.js',
+            ]
+        )
+    }}
+{% endblock %}
+
+
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/layout.ui.html.twig]: Extra lines

File: src/Pelagos/Bundle/AppBundle/Resources/views/Account/PasswordReset.html.twig [hunk 2]
```diff
@@ -23,7 +15,7 @@
                         <input type="email" size="100" name="emailAddress" id="emailAddress" required>
                         <label style="display:inline;" for="emailAddress" class="error"></label>
                         <p>
-                            <button type="submit">Reset</button>
+                            <button class="btn btn-dark" type="submit">Reset</button>
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Account/PasswordReset.html.twig]: Can remove style for this button, no other buttons have style

File: src/Pelagos/Bundle/AppBundle/Resources/views/template/AdminOnly.html.twig [hunk 3]
```diff
@@ -1,4 +1,5 @@
-<div class="messages error">
+<div class="messages error alert alert-danger">
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/template/AdminOnly.html.twig]: Again, not related to this pull request. 
Non-DRPM users are able to access DRPM pages

File: src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig [hunk 4]
```diff
@@ -1,13 +1,10 @@
-{% extends "PelagosAppBundle:template:UI.html.twig" %}
-
-{% block head %}
-    {{ parent() }}
-{% endblock %}
+{% extends "PelagosAppBundle::layout.ui.html.twig" %}
 
 {% block body %}
-    <div class="messages error">
+    <div class="messages error alert alert-warning">
```
Comment (fightingtexasaggie) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig]: Should this be an error or warning (red)?

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig]: I mixed it up here and there, we should probably revisit these messages, maybe refactor a thing or two. In this case I felt danger was a bit harsh.

File: src/Pelagos/Bundle/AppBundle/Resources/views/MdApp/upload-complete.html.twig [hunk 5]
```diff
@@ -2,6 +2,8 @@
 
 {% block head %}
     {{ parent() }}
+
```
Comment (fightingtexasaggie) [inline src/Pelagos/Bundle/AppBundle/Resources/views/MdApp/upload-complete.html.twig]: The upload from MDAPP behaviour has been replaced by the dataset-review process. At some point we should probably refactor and remove these vestigial stubs.

**Pull request merged**